src/app/shared/translate/missing-translation.helper.ts

```diff
@@ -0,0 +1,10 @@
+import { MissingTranslationHandler, MissingTranslationHandlerParams } from '@ngx-translate/core';
+
+export class MissingTranslationHelper implements MissingTranslationHandler {
```

Comment:
Could we add TypeDocs to this new class?

Reply:
@tdonohue 
I added the missing TypeDoc.

If I understand correctly, you would like the facets to hide when there is only a single option. I experimented with this by changing the 'available value threshold' of when to show facets from 1 to 2. Unfortunately, this means that if you perform a very specific search, there will be no filters available at all. In my opinion this looks pretty strange.
![image](https://user-images.githubusercontent.com/11385967/76778250-c924fd80-67a9-11ea-9daa-b2ae9b3e69b2.png)

Reply:
@LotteHofstede : thanks for the explanation and looking into this.  This can be left as-is for now. I don't feel strongly enough about it, so I'm fine with the filters remaining even if only one option exists.